reduce/reduce.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# -------------------------------------
+# ClangBuiltLinux/misc-scripts/reduce
+# Author: Justin Stitt <justinstitt@google.com
+# As Part Of: ClangBuiltLinux and Google LLC
+# License: Apache v2.0 -- see ./LICENSE
+# -------------------------------------
+
+import argparse
+from collections.abc import Callable
+import sys
+
+from src import flags, prep
+
+
+VERSION = "0.2.1"
+
+ENTRY_POINTS: dict[str, Callable] = {"prep": prep.main, "flags": flags.main}
```

Comment:
I don't see `ENTRY_POINTS` used anywhere?

Reply:
Dead code. Was used pre-refactor, now this behavior is supplied by subparsers. Thanks! Fixed.